assets/app/scripts/controllers/overview.js

```diff
@@ -58,6 +71,99 @@ angular.module('openshiftConsole')
       console.log("podsByServiceByLabel", $scope.podsByServiceByLabel);      
     };
 
+    var podRelationships = function() {
+      $scope.servicePodsNoDeployment = {};
+      $scope.solitaryPods = {};
+
+      angular.forEach($scope.pods, function(pod, name){
+        var replicationcontrollers = [];
+        var services = [];
+        angular.forEach($scope.deployments, function(deployment, depName){
+          var ls = new LabelSelector(deployment.spec.selector);
+          if (ls.matches(pod)) {
+            replicationcontrollers.push(depName);
+          }
+        });
+        angular.forEach($scope.services, function(service, svcName){
+          var ls = new LabelSelector(service.spec.selector);
+          if (ls.matches(pod)) {
+            services.push(svcName);
+            if (replicationcontrollers.length == 0) {
```

Comment:
not quite right... the pod could be in a replication controller, but one that is not in this service, so it should appear under this service, but in `servicePodsNoDeployment`